src/cache/main.go

```diff
@@ -121,8 +121,11 @@ func main() {
 			return
 		}
 
-		// Fetch TTL from config file and convert it into a time.Duration in seconds
-		ttl := time.Duration(int(config["ttl"].(float64))) * time.Second
+		ttl := requestBody.ttl
```

Comment:
We will need to ensure this value is converted to a `time.Duration` object in seconds

Reply:
Does that mean creating a Time.duration object using seconds?